packages/parse5-plain-text-conversion-stream/lib/index.ts

```diff
@@ -0,0 +1,39 @@
+import type { ParserOptions } from 'parse5/dist/parser/index.js';
```

Comment:
whats going on with these paths? do we  not export them at the root? i.e. `'parse5'`

similarly, the tree adapter interface has no extension but these do